src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.domain.Cars;
+import racingcar.domain.RacingGame;
+import racingcar.domain.strategy.RandomMoveStrategy;
+import racingcar.view.RacingCarInput;
+import racingcar.view.RacingCarView;
+
+public class RacingController {
+
+    public void start() {
+        List<String> carNames = RacingCarInput.getCarName();
+        int endRaceCount = RacingCarInput.getRaceCount();
+
+        RacingGame racingGame = initRacingGame(carNames, endRaceCount);
```

Comment:
메서드로 추출할 필요 없이, 바로 RacingGame 객체를 생성해도 될 것 같아요! 😃 

```suggestion
        RacingGame racingGame = new RacingGame(Cars.of(carNames), endRaceCount, new RandomMoveStrategy());
```